cpp/mg_utility/mg_utils.hpp

```diff
@@ -53,7 +56,8 @@ void CreateGraphEdge(mg_graph::Graph<TSize> *graph, mgp_vertex *vertex_from, mgp
   graph->CreateEdge(memgraph_id_from, memgraph_id_to, memgraph_edge_id, graph_type);
 }
 
-///@brief Creates a weighted edge within GraphView. Requires Memgraph’s UIDs for its endpoints.
+///@brief Creates a weighted edge within GraphView. Requires Memgraph’s UIDs for
+/// its endpoints.
```

Comment:
Is this file correctly formatted?

Reply:
Fixed formatting (the snippet in question has been removed in the meantime btw).